workflows/codefresh-csdp/versions/1.1.7/images/report-image-info/src/outputs.js

```diff
@@ -1,8 +1,9 @@
-const fs = require('fs')
-
 // Outputs are used to provide some data about the template execution.
 // They are used by 3rd-party CI image-reporting feature.
 
+const fs = require('fs')
+const { OUTPUTS } = require('@codefresh-io/cf-docker-images')
```

Comment:
I don't think we need to store outputs in the package. They can be moved here and the values can be the NOTIFICATIONS enum values.